### PR TITLE
Add universal Lispworks support

### PR DESCRIPTION
**Pull request by calsys456**
Using `fast-directory-files` facility.

Seems working well on my machine...

Thank you!

### CODE REVIEW

**Comment by ruricolist** [comment]
Looking at the LispWorks documentation I linked to before (https://www.lispworks.com/documentation/lw80/lw/lw-common-lisp-18.htm#common-lisp_marker-line-1630) it seems major version features are removed on the next release, so this will revert to the old approach if run on future major versions of LispWorks (9 or greater). 

I think the correct guards would be:

For the new way:
```
#-(or lispworks4 lispworks5 lispworks6 lispworks7)
```

For the old way:

```
#+(or lispworks4 lispworks5 lispworks6 lispworks7)
```

**Comment by calsys456** [comment]
Thanks for your correction! I use more precise feature expression this time, and I hope it won't makes trouble any more :D

File: trivial-file-size.lisp [hunk 1]
```diff
@@ -46,6 +46,21 @@ Some platforms (e.g. ABCL) may return 0 when the file does not exist."
           #+(and lispworks unix)
           (sys:file-stat-size (sys:get-file-stat namestring))
 
+          #+(and lispworks (not (or unix lispworks4 lispworks5 lispworks6 lispworks7)))
```
Comment (ruricolist) [inline trivial-file-size.lisp]: I think this condition is wrong: it should be (and lispworks (not unix) (or ...)).

**Comment by calsys456** [comment]
Yes thankyou ///////
I've fixed it!

**Comment by ruricolist** [comment]
Merged, thanks!

**Pull request merged**